packages/webcomponents/src/components/payments-list/payments-list-core.tsx

```diff
@@ -2,9 +2,11 @@ import { Component, h, Prop, State, Watch, Event, EventEmitter } from '@stencil/
 import { PagingInfo, Payment, pagingDefaults } from '../../api';
 import { MapPaymentStatusToBadge, formatCurrency, formatDate, formatTime } from '../../utils/utils';
 import { ComponentError } from '../../api/ComponentError';
-import StyledHost from '../../utils/styled-host/styled-host';
-import { EmptyState, ErrorState, LoadingState } from '../table/utils';
 import { tableExportedParts } from '../table/exported-parts';
+import { StyledHost } from '../../ui-components';
+import { TableErrorState } from '../../ui-components/table-error-state';
+import { TableLoadingState } from '../../ui-components/table-loading-state';
```

Comment:
One more thing! 

Would you also import the new TableState components into `ui-components/index.ts` so that all the ui components can be imported from a single source? :)

Reply:
Good suggestion!